tutorials/39_Embedding_Metadata_for_Improved_Retrieval.ipynb

```diff
@@ -0,0 +1,410 @@
+      "model.safetensors: 100%|██████████| 133M/133M [02:41<00:00, 826kB/s]  \n",
+      "tokenizer_config.json: 100%|██████████| 366/366 [00:00<00:00, 1.92MB/s]\n",
+      "vocab.txt: 100%|██████████| 232k/232k [00:00<00:00, 1.09MB/s]\n",
+      "tokenizer.json: 100%|██████████| 711k/711k [00:00<00:00, 1.11MB/s]\n",
+      "special_tokens_map.json: 100%|██████████| 125/125 [00:00<00:00, 374kB/s]\n",
+      "1_Pooling/config.json: 100%|██████████| 190/190 [00:00<00:00, 596kB/s]\n",
+      "Batches: 100%|██████████| 2/2 [00:40<00:00, 20.03s/it]\n",
+      "Batches: 100%|██████████| 2/2 [00:38<00:00, 19.09s/it]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'writer': {'documents_written': 47}}"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import wikipedia\n",
+    "from haystack import Document\n",
+    "from haystack.document_stores.in_memory import InMemoryDocumentStore\n",
+    "\n",
+    "some_bands=\"\"\"The Beatles,The Cure\"\"\".split(\",\")\n",
+    "\n",
+    "raw_docs=[]\n",
+    "\n",
+    "for title in some_bands:\n",
+    "    page = wikipedia.page(title=title, auto_suggest=False)\n",
+    "    doc = Document(content=page.content, meta={\"title\": page.title, \"url\": page.url})\n",
+    "    raw_docs.append(doc)\n",
+    "\n",
+    "document_store = InMemoryDocumentStore(embedding_similarity_function=\"cosine\")\n",
+    "document_store_with_embedded_metadata = InMemoryDocumentStore(embedding_similarity_function=\"cosine\")\n",
+    "\n",
+    "indexing_pipeline = create_indexing_pipeline(document_store=document_store)\n",
+    "indexing_with_metadata_pipeline = create_indexing_pipeline(document_store=document_store_with_embedded_metadata, metadata_fields_to_embed=[\"title\"])\n",
+    "\n",
+    "indexing_pipeline.run({\"cleaner\":{\"documents\": raw_docs}})\n",
+    "indexing_with_metadata_pipeline.run({\"cleaner\":{\"documents\": raw_docs}})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Comparing Retrieval With and Without Embedded Metadata\n",
+    "\n",
+    "As a final step, we will be creating a retrieval pipeline that will have 2 retrivers:\n",
+    "- First: retrieving from the `document_store`, where we have not embedded metadata.\n",
+    "- Second: retireving from the `document_store_with_embedded_metadata`, where we have embedded metadata.\n",
+    "\n",
+    "We will then be able to compare the restults and see if embedding metadata has helped with retrieval in this case.\n",
+    "\n",
+    "> 💡 Here, we are using the `InMemoryEmbeddintRetriever` because we used the `InMemoryDocumentStore` above. If you're using another documet store, change this to use the accompanying embedding retriever for the document store you are using. Check out the [Embedders Documentation](https://docs.haystack.deepset.ai/v2.0/docs/embedders) for a full list"
```

Comment:
```suggestion
    "> 💡 Here, we are using the `InMemoryEmbeddintRetriever` because we used the `InMemoryDocumentStore` above. If you're using another document store, change this to use the accompanying embedding retriever for the document store you are using. Check out the [Embedders Documentation](https://docs.haystack.deepset.ai/v2.0/docs/embedders) for a full list"
```